addons/radios/XEH_postInitClient.sqf

```diff
@@ -6,6 +6,12 @@
     _vehicle setVariable [QGVAR(source), _source];
 }] call CBA_fnc_addEventHandler;
 
+[QGVAR(stop), {
+    params ["_vehicle", "_source"];
+    deleteVehicle _source;
```

Comment:
Check if `_source` exists, it will not for JIP clients.

Reply:
That's done just before the event is called in `fnc_stop`

Reply:
No, it should be on each client, as some might have it, some not.